src/pudl/transform/ferc1.py

```diff
@@ -548,8 +548,8 @@ class Config:
     rename_columns_ferc1: RenameColumnsFerc1 = RenameColumnsFerc1(
         dbf=RenameColumns(),
         xbrl=RenameColumns(),
-        xbrl_instant=RenameColumns(),
-        xbrl_duration=RenameColumns(),
+        instant_xbrl=RenameColumns(),
+        duration_xbrl=RenameColumns(),
```

Comment:
Eeek, we had this inconsistency with all of the actual parameters and it didn't cause any problems? That's kind of amazing.